### DISCUSSION TITLE
阴

### DISCUSSION THREAD

**Discussion by yileinafs** [share121/inter-knot#1980] [其他]
![bdfdfe6800f2ce83e1a564a644aee9c42143018054192409310](https://github.com/user-attachments/assets/9a674173-ea4e-400a-8839-f15a30f6d9ce)
不是，死水还阴啊?

**Comment by bbdqz** [comment]
布什,戈门

**Comment by bai-shengjiang** [comment]
不是，怎么这里也有村里人

**Comment by demolute** [comment]
中规中矩吧。火影现在这环境，死水能打，但牌面不至于阴间。
